projects/igniteui-angular/src/lib/splitter/splitter-bar/splitter-bar.component.ts

```diff
@@ -198,6 +198,11 @@ export class IgxSplitBarComponent {
         return !!relatedTabs.find(x => x.resizable === false);
     }
 
+    protected get panesHidden() {
```

Comment:
This looks very similar to resizeDisallowed. Check can be moved there. There does not seem to be need of an additional getter.